client/components/mma/accountoverview/ProductCard.tsx

```diff
@@ -385,7 +381,7 @@ export const ProductCard = ({
 							</h4>
 							<div>
 								<dl css={keyValueCss}>
-									<dt>{atob('TFBQRlJFRTZHTFRY')}</dt>
+									<dt>{window.atob('TFBQRlJFRTZHTFRY')}</dt>
```

Comment:
the linter seems to get confused between the browser and node versions of this function, the node one version throwing a deprecation warning. This should force the hand of the linter to let it know that we are only concerned with the browser implimentation here